mobile/library/cc/engine_builder.cc

```diff
@@ -734,6 +739,15 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
     rcv_buf_sock_opt->set_int_value(socket_receive_buffer_size_);
```

Comment:
change this to datagram only?

Reply:
Initially I wanted to do it in a separate PR, but the change is small enough that I can do it in this PR. Done.